tests/fugue_sql/test_workflow_parse.py

```diff
@@ -515,6 +515,17 @@ def test_drop():
         dag,
     )
 
+def test_fill():
+    dag = FugueWorkflow()
+    a = dag.df([[None, 1], [1, None]], "a:int, b:int")
+    b = a.fillna({"a": 99, "b": -99})
+    assert_eq(
+        """
+    a=create [[NULL, 1],[1, NULL]] schema a:int, b:int
+    FILL NULLS PARAMS a:99, b:-99 FROM a
+    """,
```

Comment:
Perfect, can you also anonymous case?

```
create [[NULL, 1],[1, NULL]] schema a:int, b:int
FILL NULLS PARAMS a:99, b:-99
```
this should also equal to your line 520 521